compiler/src/iree/compiler/Codegen/Common/LowerMicroKernelsToCalls.cpp

```diff
@@ -0,0 +1,61 @@
+// Copyright 2023 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "iree/compiler/Codegen/Interfaces/MicroKernelOpInterface.h"
+#include "iree/compiler/Codegen/PassDetail.h"
+#include "iree/compiler/Codegen/Passes.h"
+#include "mlir/Dialect/MemRef/IR/MemRef.h"
+#include "mlir/Transforms/GreedyPatternRewriteDriver.h"
+
+namespace mlir {
+namespace iree_compiler {
+
+namespace {
+struct LowerMicroKernelOpsToCallsPass
+    : LowerMicroKernelOpsToCallsBase<LowerMicroKernelOpsToCallsPass> {
+  void getDependentDialects(DialectRegistry &registry) const override {
+    registry.insert<memref::MemRefDialect>();
```

Comment:
We also want func dialect because func.call op could be generated.

Reply:
Good catch. This part is tricky. It fails sometimes, doesnt fail sometimes.